validator/client/beacon-api/duties.go

```diff
@@ -220,22 +187,18 @@ func (c *beaconApiValidatorClient) dutiesForEpoch(
 		if committeeMappingKey, ok := attesterDutiesMapping[v.index]; ok {
 			committeeIndex = committeeMappingKey.committeeIndex
 			attesterSlot = committeeMappingKey.slot
-
-			if committeeValidatorIndices, ok = committeeMapping[committeeMappingKey]; !ok {
-				return nil, errors.Errorf("failed to find validators for committee index `%d` and slot `%d`", committeeIndex, attesterSlot)
-			}
+			committeeValidatorIndices = make([]primitives.ValidatorIndex, committeeMappingKey.committeeLength) // make a fake committee because only length is used
```

Comment:
I'm faking the indices here because we don't really use the actual committees anywhere for normal workflows, instead we use the committee lengths. we are using the committees for slashing however so it's not completely clear to me whether we can avoid calling the committee endpoint on epoch.